book/src/02_development/06_style_guide.md

```diff
@@ -0,0 +1,57 @@
+# Style Guide
+
+This is a style guide for the EDR project.
+
+## Procedural derive macros
+
+When deriving multiple traits, use the following rules to order them:
+
+1. Standard library traits before external crates
+2. Sub-traits before super-traits
+3. Alpabetical order
+
+For example:
+
+```rust
+#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
+```
+
+Or, for sub- and super-traits:
+
+```rust
+#[derive(PartialEq, Eq, PartialOrd, Ord)]
+```
+
+## Member ordering
+
+When adding a variant to an `enum` or a field to a `struct` or enum variant, by default follow alphabetical order. If it makes more sense to follow custom ordering, feel free to do so.
+
+### Member functions
+
+For member functions, use the following default rules to order them:
```

Comment:
"default rules" -- I'm assuming this means it's ok to deviate if it makes sense? Would be good to make it more explicit if possible

Reply:
Addressed in https://github.com/NomicFoundation/edr/pull/660/commits/d1a250f6ebd06ebdf3d9ebeaf7c172e057135d54